src/components/layout/header.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import Link from 'next/link';
+
+import Avatar from '@/components/ui/avatar';
+import Logo from '@/components/ui/logo';
+
+import { useUserStore } from '@/store/user';
+
+export default function Header() {
+  const { user } = useUserStore();
+
+  return (
+    <header className="flex w-full items-center justify-between border-b border-gray-100 bg-white px-6 py-2">
+      <Link href="/">
+        <Logo />
+      </Link>
+      {user?.profileUrl ? (
+        <Avatar size="default" src={user.profileUrl} alt="avatar" fallback={user.name?.charAt(0)} />
+      ) : (
+        <Avatar size="default" alt="avatar" fallback={user?.name?.charAt(0)} />
+      )}
```

Comment:
이거 굳이 분기를 이렇게 복잡하게 할 필요가 있을까요??

```suggestions
<Avatar src={user.profileUrl ?? ''} alt={user.name} fallback={user.name?.charAt(0)} />
```

이정도로만 넘겨주어도 되지 않을까요?